.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,8 @@
 ## Description
 
-Please provide a short description of the change being proposed.
+<!-- TODO: Delete this comment and replace it with a meaningful description of what this change will do, or is for. Bonus points for linking back to related issues or PRs!  -->
+
+<!-- Please note that by not including a description, you are asking reviewers to do extra work to understand the context of this change, which may lead to your PR taking much longer to review, or result in it not being reviewed at all. -->
```

Comment:
I like the idea, but wonder if we don't want to have it as text rather than as a comment. My reasoning is that the comment below, above the checklist, is aimed at the maintainers of the repository, rather than at people submitting pull requests. Thoughts?

Reply:
That's a fair point RE: different audiences for the template. I suppose the biggest risk in this being an 'invisible' comment is that people can effectively ignore it, since it will only be visible to them in edit mode. If we change it to text, and the user makes no alterations before submitting, it will be much more obvious that they've explicitly chosen to ignore the guidance around descriptions.

Reply:
@jdhmtl Updated the content to use visible text rather than hidden comments

Reply:
Thanks!

I really like the idea of explaining the value of the detailed comment rather than just "please do this because I said so."